test/SlowTests/Issues/RavenDB-17597.cs

```diff
@@ -0,0 +1,172 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using FastTests;
+using FastTests.Client;
+using NuGet.ContentModel;
+using Orders;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Documents.Operations.Indexes;
+using Raven.Client.Documents.Session;
+using Raven.Client.Exceptions.Database;
+using Raven.Client.Exceptions.Documents.Indexes;
+using Raven.Client.ServerWide;
+using Raven.Client.ServerWide.Operations;
+using Raven.Server;
+using Raven.Server.Commercial;
+using Raven.Server.Config;
+using Raven.Server.Documents;
+using Raven.Server.Documents.Indexes;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_17597 : ClusterTestBase
+    {
+        public RavenDB_17597(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task ModifyTaskIndexTest()
```

Comment:
Let's add another test but with a non-faulty side-by-side index.
Simply stop the index with `StopIndexOperation`, modify it and restart the server.

Reply:
Done